files/image_config/rsyslog/rsyslog.d/00-sonic.conf

```diff
@@ -0,0 +1,41 @@
+## Quagga rules
+
+if $programname == ["quagga",
+                    "watchquagga",
+                    "zebra"]
+    then {
+    /var/log/quagga/zebra.log
+    stop
+}
+
+if $programname == "bgpd" then {
+    /var/log/quagga/bgpd.log
+    stop
+}
+
+## Teamd rules
+
+if $programname contains "teamd_" then {
+    /var/log/teamd.log
+    stop
+}
+
+## SONiC rules
+
+if $programname == ["fpmsyncd",
+                    "intfsyncd",
+                    "neighsyncd",
+                    "orchagent",
+                    "portsyncd",
+                    "saidump",
+                    "saiplayer",
+                    "syncd",
+                    "swssconfig",
+                    "swssloglevel",
+                    "teamsyncd",
+                    "updategraph"]
+    then {
+    ?SONiCPerProcessLogfile,SONiCFileFormat
```

Comment:
Per-process SONiC logging is not currently working. I updated the list of programs, just in case we want it working, but I believe we'd just like everything to log to /var/log/syslog. If so, I will remove this. Please share your thoughts.